projects/igniteui-angular/package.json

```diff
@@ -66,25 +66,25 @@
         "csv export"
     ],
     "dependencies": {
-        "@types/hammerjs": "^2.0.36",
+        "@types/hammerjs": "^2.0.40",
         "hammerjs": "^2.0.8",
         "jszip": "^3.6.0",
         "tslib": "^2.0.0",
         "@juggle/resize-observer": "^3.3.0",
         "igniteui-trial-watermark": "^1.0.3",
         "lodash.mergewith": "^4.6.2",
         "uuid": "^8.3.0",
-        "@igniteui/material-icons-extended": "^2.7.0"
+        "@igniteui/material-icons-extended": "^2.8.0"
     },
     "peerDependencies": {
-        "@angular/common": "^12.0.0",
-        "@angular/core": "^12.0.0",
-        "@angular/animations": "^12.0.0",
-        "@angular/forms": "^12.0.0",
+        "@angular/common": "^12.1.1",
+        "@angular/core": "^12.1.1",
+        "@angular/animations": "^12.1.1",
+        "@angular/forms": "^12.1.1",
         "web-animations-js": "^2.3.2"
     },
     "igxDevDependencies": {
-        "@igniteui/angular-schematics": "~12.0.800"
+        "@igniteui/angular-schematics": "~12.1.810-beta.0"
```

Comment:
Why not using [12.0.803](https://www.npmjs.com/package/@igniteui/angular-schematics/v/12.0.803)